index.html

```diff
@@ -1,36 +1,199 @@
+					don’t hestiate to contact me.</p>
+				<h3 class="connect">LET’S CONNECT</h3>
+				<ul class="social-media">
+
+					<li class="sm"><a href="https://twitter.com/PRATAP_TWT"><img src='images/twitter.svg' width='20'
+								height='20' alt="Twitter Profile" title=''></a></li>
+					<li class="sm"><a href="https://www.linkedin.com/in/pratap-kumar-panabaka-755489236/"><img
+								src='images/linkedin.svg' width='20' height='20' alt="LinkedIn Profile" title=''></a>
+					</li>
+					<li class="sm"><a href=""><img src='images/m.svg' width='20' height='20' alt='' title=''></a></li>
+					<li class="sm"><a href="https://github.com/PRATAP-KUMAR"><img src='images/github.svg' width='20'
+								height='20' alt="GitHub Profile" title=''></a></li>
+					<li class="sm"><a href=""><img src='images/v.svg' width='20' height='20' alt='' title=''></a></li>
+
+				</ul>
+			</section>
+		</div>
+		<section class='projects-section'>
+			<div class="project-wrapper grid-box">
+				<div class="project">
+					<img src="projects/project1.png" alt="project 1" title="Project 1" width="295" height="220" />
+					<div class="project-heading">
+						<h2 class="project-title">Project 1</h2>
+						<div class="project-meta-data">
+							<div class="project-client">Linux</div>
+							<img src='images/dot.png' width='8' height='8' alt="Separator Dot" title=''>
+							<div class="project-tech">Linux User</div>
+							<img src='images/dot.png' width='8' height='8' alt="Separator Dot" title=''>
+							<div class="project-year">2015</div>
+						</div>
+						<p class="project-description">
+							A daily selection of privately personalized reads; no accounts or sign-ups required.
+						</p>
+						<ul class="project-tags">
+							<li class="tags">html</li>
+							<li class="tags">css</li>
+							<li class="tags">javaScript</li>
+						</ul>
+						<div class="project-button">
+							<button type="button">See Project</button>
+						</div>
+					</div>
+				</div>
+
+				<div class="project">
+					<img src="projects/project2.png" alt="project 2" title="Project 2" width="295" height="220" />
+					<div class="project-heading">
+						<h2 class="project-title">Project 2</h2>
+						<div class="project-meta-data">
+							<div class="project-client">Gnome</div>
+							<img src='images/dot.png' width='8' height='8' alt="Separator Dot" title=''>
+							<div class="project-tech">Gnome User</div>
+							<img src='images/dot.png' width='8' height='8' alt="Separator Dot" title=''>
+							<div class="project-year">2018</div>
+						</div>
+						<p class="project-description">
+							A daily selection of privately personalized reads; no accounts or sign-ups required.
+						</p>
+						<ul class="project-tags">
+							<li class="tags">html</li>
+							<li class="tags">css</li>
+							<li class="tags">javaScript</li>
+						</ul>
+						<div class="project-button">
+							<button type="button">See Project</button>
+						</div>
+					</div>
+				</div>
```

Comment:
- [ ] [OPTIONAL]  Great job so far but as I can see here the padding  is not the same as the Figma design so please Fix it to be the same as much as possible. 👌🏻👌🏻